bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/help/view/HelpNavigationBarBottomAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.help.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import net.gini.android.bank.sdk.databinding.GbsHelpNavigationBarBottomBinding
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface HelpNavigationBarBottomAdapter: InjectedViewAdapter {
```

Comment:
[Request] Please rename it to `DigitalInvoiceHelpNavigationBarBottomAdapter` including the default implementation and property name in GiniBank. I know it's looong (that's what she said!) but it makes it really obvious that it's different from the Capture SDK's `HelpNavigationBarBottomAdapter`.